src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/NotifyServiceTest.kt

```diff
@@ -108,7 +108,7 @@ class NotifyServiceTest {
       Pair("comFullName", "Joe Bloggs"),
       Pair("offenderFullName", "James Jonas"),
       Pair("crn", "X11111"),
-      Pair("dateDescriptions", "[Licence start date, Licence end date]"),
+      Pair("dateDescriptions", listOf("Licence start date", "Licence end date")),
```

Comment:
Now a list of strings is passed into the template, rather than a list as a single string

Reply:
Not sure about this one. Think it needs to be a single string? Or have you checked the result in the email?

Reply:
This is how its done for the licence prompt